src/SPC/store/scripts/zig-cc.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
-BUILDROOT_ABS="$(realpath "$SCRIPT_DIR/../../buildroot/include" 2>/dev/null || true)"
+BUILDROOT_ABS="$(realpath "$SCRIPT_DIR/../../../buildroot/include" 2>/dev/null || true)"
```

Comment:
How about using `BUILD_INCLUDE_PATH` variable? We call `zig-cc.sh` inside SPC can use it.

Reply:
That would make it a problem with docker. or when the env variable was set at oen time but then unset at another We need to use relative paths unfortunately.